### PR TITLE
add violation

### PR DESCRIPTION
**Pull request by seanpdoyle**

### CODE REVIEW

File: addon/extend-qunit.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import QUnit from 'qunit';
 
-function normalizeString(actual) {
-  if (!actual) {
+function normalizeString(actual){
```
Comment (seanpdoyle) [inline addon/extend-qunit.js]: One (or more) spaces required before opening brace for block expressions

File: addon/extend-qunit.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+  if (!actual){
```
Comment (seanpdoyle) [inline addon/extend-qunit.js]: One (or more) spaces required before opening brace for block expressions

File: addon/scope.js [hunk 3]
```diff
@@ -18,7 +18,8 @@ function defineTestHelpers(context) {
 
 function blacklistTestHelpers(context) {
   BLACKLISTED_HELPERS.forEach(name => {
-    context[name] = function() {
+    if(true) console.log('hi');
```
Comment (seanpdoyle) [inline addon/scope.js]: If statement without curly braces<br>One space required after "if" keyword

File: addon/within.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 import Scope from './scope';
 
-Ember.Test.registerHelper('within', function(app, selector, block) {
+Ember.Test.registerHelper('within', function(app,selector,block) {
```
Comment (seanpdoyle) [inline addon/within.js]: One space required before function parameter 'selector'<br>One space required before function parameter 'block'

File: addon/scope.js [hunk 5]
```diff
@@ -29,7 +30,7 @@ function blacklistTestHelpers(context) {
 }
 
 class Scope {
-  constructor(app, selector) {
+  constructor(app,selector) {
```
Comment (seanpdoyle) [inline addon/scope.js]: One space required before function parameter 'selector'

File: addon/scope.js [hunk 6]
```diff
@@ -29,14 +30,14 @@ function blacklistTestHelpers(context) {
...
     this.app = app;
     this.selector = selector;
     defineTestHelpers(this);
     blacklistTestHelpers(this);
   }
 
-  within(selector, block, ...args) {
+  within(selector,block, ...args) {
```
Comment (seanpdoyle) [inline addon/scope.js]: One space required before function parameter 'block'

File: addon/scope.js [hunk 7]
```diff
@@ -29,14 +30,14 @@ function blacklistTestHelpers(context) {
...
-  constructor(app, selector) {
+  constructor(app,selector) {
     this.app = app;
     this.selector = selector;
     defineTestHelpers(this);
     blacklistTestHelpers(this);
   }
 
-  within(selector, block, ...args) {
+  within(selector,block,...args) {
```
Comment (seanpdoyle) [inline addon/scope.js]: One space required before function parameter 'block'<br>One space required before function parameter '...'

File: addon/scope.js [hunk 8]
```diff
@@ -29,15 +30,15 @@ function blacklistTestHelpers(context) {
...
+  constructor(app,selector) {
     this.app = app;
     this.selector = selector;
     defineTestHelpers(this);
     blacklistTestHelpers(this);
   }
 
-  within(selector, block, ...args) {
-    return block(new Scope(this.app, `${this.selector} ${selector}`, ...args));
+  within(selector,block,...args) {
```
Comment (seanpdoyle) [inline addon/scope.js]: One space required before function parameter 'block'<br>One space required before function parameter '...'

File: addon/scope.js [hunk 9]
```diff
@@ -10,7 +10,7 @@ const BLACKLISTED_HELPERS = [
 function defineTestHelpers(context) {
   Object.keys(context.app.testHelpers).forEach(name => {
     const helper = context.app.testHelpers[name];
-    context[name] = function(selector, ...args) {
+    context[name] = function(selector,...args) {
```
Comment (seanpdoyle) [inline addon/scope.js]: One space required before function parameter '...'

File: tests/dummy/config/environment.js [hunk 10]
```diff
@@ -19,7 +19,7 @@ module.exports = function(environment) {
     }
   };
 
-  if (environment === 'development') {
+  if(environment === 'development') {
```
Comment (seanpdoyle) [inline tests/dummy/config/environment.js]: One space required after "if" keyword

File: tests/dummy/config/environment.js [hunk 11]
```diff
@@ -27,7 +27,7 @@ module.exports = function(environment) {
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
   }
 
-  if (environment === 'test') {
+  if(environment === 'test') {
```
Comment (seanpdoyle) [inline tests/dummy/config/environment.js]: One space required after "if" keyword

File: tests/dummy/config/environment.js [hunk 12]
```diff
@@ -39,7 +39,7 @@ module.exports = function(environment) {
     ENV.APP.rootElement = '#ember-testing';
   }
 
-  if (environment === 'production') {
+  if(environment === 'production') {
```
Comment (seanpdoyle) [inline tests/dummy/config/environment.js]: One space required after "if" keyword

File: tests/dummy/config/environment.js [hunk 13]
```diff
@@ -39,7 +39,7 @@ module.exports = function(environment) {
     ENV.APP.rootElement = '#ember-testing';
   }
 
-  if (environment === 'production') {
+  if(environment === 'production'){
```
Comment (seanpdoyle) [inline tests/dummy/config/environment.js]: One space required after "if" keyword<br>One (or more) spaces required before opening brace for block expressions

File: tests/dummy/config/environment.js (hunk 10 above)
Comment (houndci-staging) [inline tests/dummy/config/environment.js:22]: One space required after "if" keyword

File: tests/dummy/config/environment.js (hunk 13 above)
Comment (houndci-staging) [inline tests/dummy/config/environment.js:42]: One space required after "if" keyword<br>One (or more) spaces required before opening brace for block expressions

**Pull request closed without merging**